.github/actions/check-pr/dist/licenses.txt

```diff
@@ -0,0 +1,695 @@
+@actions/core
```

Comment:
[q] where do these `licenceses.txt` files in the dist come from? If they are auto generated mention it perhaps in the markdown? Are they mandatory?

Reply:
That's generated by ncc via the `--license licenses.txt` flag. Technically this is not mandatory, and maybe we don't need it here but it feels correct to me to mention the licenses of dependencies we vendor into the js file we have in our repo.